<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindow</class>
 <widget class="QDialog" name="ProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Profile Info</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Profile Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="profileNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="gameDirectoryCheckbox">
       <property name="text">
        <string>Game Directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="gameDirectoryEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="gameDirectoryBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="resolutionEditX"/>
       </item>
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string notr="true">x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="resolutionEditY"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="resolutionCheckbox">
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="launcherVisibilityCheckbox">
       <property name="text">
        <string>Launcher Visibility:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="launcherVisibilityCombobox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Version Select</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableSnapshots">
     <property name="text">
      <string>Enable experimental development Versions (&quot;snapshots&quot;)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableBeta">
     <property name="text">
      <string>Allow use of old &quot;Beta&quot; Minecraft Versions (From 2010-2011)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableAlpha">
     <property name="text">
      <string>Allow use of old &quot;Alpha&quot; Minecraft Versions (From 2010)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Use Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="versionSelectCombobox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Java Settings (Advanced)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="executableEdit">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="executableButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="jvmArgumentsEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="executableCheckbox">
       <property name="text">
        <string>Executable:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="jvmArgumentsCheckbox">
       <property name="text">
        <string>JVM Arguments:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="serverCheckbox">
     <property name="text">
      <string>Connect to Server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server IP:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Server Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="portEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="demoModeCheckbox">
     <property name="text">
      <string>Start in demo mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="disableMultiplayerCheckBox">
     <property name="text">
      <string>Disable Multiplayer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="disableChatCheckBox">
     <property name="text">
      <string>Disable Chat</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="minecraftOptionsCheckBox">
       <property name="text">
        <string>Additional Options:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minecraftOptionsEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="gameModeCheckBox">
     <property name="text">
      <string>Use Gamemode</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createShortcutButton">
       <property name="text">
        <string>Create Shortcut</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openGameDirectoryButton">
       <property name="text">
        <string>Open Game Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveProfileButton">
       <property name="text">
        <string>Save Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
